### PR TITLE
Atomic repo clean up

### PR DESCRIPTION
**Pull request by andresgalante**
Hi!

This PR is a clean up:

- It empties the templates for PRs and issues
- Creates a new READ ME
- Adds license file
- Adds license headers to all our scss
- Moves core CSS to `patternfly.scss`

I've also already removed the branches we don't use.

@srambach @jgiardino @bleathem can you please review it? if you want we can have a small meeting and I sow you the changes.

@LHinson Can you please read the first part of the READ ME? it's here https://github.com/andresgalante/patternfly-atomic/blob/clean-up/README.md

Thanks a lot guys! 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,192 @@
-# Pattern Lab Node - Gulp Edition
+# PatternFly 4 CSS
 
-The Gulp wrapper around [Pattern Lab Node Core](https://github.com/pattern-lab/patternlab-node) providing tasks to interact with the core library and move supporting frontend assets.
+**PatternFly 4** is the best open source modular design system you'll ever use to build enterprise web applications.
+
+Think of PatternFly as a "skinned" version of [Bootstrap v4](http://v4-alpha.getbootstrap.com/) with additional components and customizations.
+
+This repository is javascript free, you'll find only find CSS here.
```
Comment (jgiardino) [inline README.md]: Minor typo here: "you'll ~~find~~ only find CSS here."

File: README.md [hunk 2]
```diff
@@ -1,64 +1,192 @@
...
+You can contact the patternFly community on the [PatternFly mailing list](https://www.redhat.com/mailman/listinfo/patternfly) or chat with us on the [PatternFly slack channel](https://patternfly.slack.com/).
+
+
+## Contributing to PatternFly 4
+
+PatternFly 4 is implemented using [Pattern Lab Node - Gulp Edition](https://github.com/pattern-lab/patternlab-node).
+
+### Coding style guidelines
+
+Please read the coding style guidelines for PatternFly 4.
```
Comment (jgiardino) [inline README.md]: Were you going to add a link to the guidelines here?

File: README.md [hunk 3]
```diff
@@ -1,64 +1,192 @@
...
+To generate the front-end for Pattern Lab type:
+
+```
+npm run gulp patternlab:build
+```
+
+#### Watch for changes and re-generate Pattern Lab
 
-## Packaged Components
+To watch for changes, re-generate the front-end, and server it via a BrowserSync server,  type:
```
Comment (jgiardino) [inline README.md]: Just a minor typo: "and serve ~~r~~ it via a..."

File: README.md [hunk 4]
```diff
@@ -1,64 +1,192 @@
...
+
+```
+npm run gulp patternlab:serve
+```
+
+BrowserSync should open [http://localhost:3000](http://localhost:3000) in your browser.
+
+___
+
+Built with :heart: by the [PattrenFly Community](https://www.patternfly.org/community/) and supported by the Red Hat UXD Team.
```
Comment (jgiardino) [inline README.md]: Doh: "PattrenFly"

File: README.md [hunk 5]
```diff
@@ -1,64 +1,192 @@
...
+
```
Comment (jgiardino) [inline README.md]: What do you think about adding the command for building the github page here?

File: source/scss/_pf-shame.scss [hunk 6]
```diff
@@ -1,4 +1,19 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// Patternfly Shame
+// --------------------------------------------------
+// The nest fot all our shamefull code.
```
Comment (jgiardino) [inline source/scss/_pf-shame.scss]: Just more types: "The nest ~~fot~~ **for** all our ~~shamefull~~ **shameful** code."

**Comment by jgiardino** [comment]
Looks great, Andres! Thanks for doing this!

File: README.md (hunk 1 above)
Comment (srambach) [inline README.md]: And the comma needs to be a semicolon :)

File: README.md [hunk 7]
```diff
@@ -1,64 +1,192 @@
...
+## Contributing to PatternFly 4
+
+PatternFly 4 is implemented using [Pattern Lab Node - Gulp Edition](https://github.com/pattern-lab/patternlab-node).
+
+### Coding style guidelines
+
+Please read the coding style guidelines for PatternFly 4.
+
+The guidelines are a work in progress and a live document, if you have any suggestion please [contact the PatternFly community](#communication-channels) or send a PR.
```
Comment (srambach) [inline README.md]: Another semicolon - or maybe even better, a period - instead of comma here. Also, "suggestions" should be plural.

**Comment by srambach** [comment]
Great improvements, @andresgalante !

**Comment by andresgalante** [comment]
@srambach @jgiardino I've fixed all the typos and added the github pages instructions. For now it's ok, but I feel we will need to flesh the instructions out to make them actually usable.

Thanks a lot for reviewing it 😄

**Comment by andresgalante** [comment]
@jgiardino @srambach @bleathem I've solved the conflicts, can you please merge this one?

**Pull request merged**